applications/_plugins/common/texture.h

```diff
@@ -107,6 +107,7 @@ extern "C" {
 
 #define CMP_FOURCC_BC6H                    CMP_MAKEFOURCC('B', 'C', '6', 'H')
 #define CMP_FOURCC_BC7                     CMP_MAKEFOURCC('B', 'C', '7', 'x')
+#define CMP_FOURCC_BC7_SRGB                CMP_MAKEFOURCC('B', 'C', '7', 's')
```

Comment:
Is this a reasonable FourCC? Does it have to be congruent to a well-known value?

Reply:
@w23  This seems reasonable, I am not aware of any other open source options that use it.